modules/ROOT/pages/architecture/hexagonale_architecture.adoc

```diff
@@ -194,6 +194,169 @@ This makes the application more difficult to understand and harder to locate the
 
 Therefore, we propose to implement the domain model as anemic entities and make usage of use cases and services to implement the business logic and interact with the domain models.
 
+== Module-based hexagonal component architecture
+The previously described approach of structuring using package structures is particularly suitable for lightweight and smaller applications.
+
+For larger applications, it makes sense to additionally implement the division using Maven or Gradle modules.
+This has the advantage that the application core of the hexagon remains completely without special technical frameworks such as JPA.
+As a result, it is technically not possible to use these functions in the core, which permanently leads to a "cleaner" core.
+
+//[[img-t-hexagonal-architecture-module]]
+//.Module-based hexagonal component architecture
+//image::hexagonal_component_architecture_modules.drawio.svg["devonfw hexagonal architecture module",scaledwidth="80%",align="center"]
+
+The division into modules looks like this:
+
+[source,plaintext]
+----
+application A1
+├── bootstrap (module)
+└── component C1 (module)
+    ├── core (module)
+    │   ├── domain
+    │   ├── port
+    │   │   ├── in
+    │   │   └── out
+    │   ├── usecase
+    │   └── service (optional)
+    └── adapter (module)
+        ├── in
+        │   └── web
+        │       └── (analog with packages)
+        └── out
+            └── repository
+                └── (analog with packages)
+----
+
+At the main level of the application there is the Bootstrap Module, which in Spring contains, for example, the configurations and SpringBootApplication.
+This module provides the starting point of the application and at the same time has a dependency on all cores and adapters of the business components.
+The business components, of which there can be any number, are also defined as modules at the main level.
+In the first step, we only assume one business module.
+This is divided into both the core module and the adapter module.
+In this way, core and adapter can define different dependencies and the core remains free of technical frameworks.
+
+The adapters in turn are divided into `in` and `out` adapters as in the package structure described above.
```

Comment:
This sentence ist kind of irritating as we're talking about dividing it further and then it's saif only on packages. I would turn it around and say that we recommend to group all adapters in one module, because reasons.
Then argue that this adapter module uses packages to structure the different adapters.